File: output/ifc4x3_dev_73740fe4/entity/IfcEdgeLoop.java
package com.topoplanet.ifc4x3_dev_73740fe4.entity;
import com.topoplanet.test.type.*;


import java.util.List;

public class IfcEdgeLoop extends IfcLoop {

    // INVERSE attributes:
    // Ne : IfcInteger := SIZEOF(EdgeList);
    // INVERSE attributes:
    // IsClosed : (EdgeList[1].EdgeStart) :=: (EdgeList[Ne].EdgeEnd);
    // IsContinuous : IfcLoopHeadToTail(SELF);

    // WHERE constraints:
    // IsClosed : (EdgeList[1].EdgeStart) :=: (EdgeList[Ne].EdgeEnd);
    // IsContinuous : IfcLoopHeadToTail(SELF);
}

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRepresentationItem/IfcTopologicalRepresentationItem/IfcLoop/IfcEdgeLoop/IfcEdgeLoop.cs
public class IfcEdgeLoop : IfcLoop
{
    public List<IfcOrientedEdge> EdgeList { get; set; }

    // === WHERE CLAUSES ===
    // IsClosed : (EdgeList[1].EdgeStart) :=: (EdgeList[Ne].EdgeEnd)
    // IsContinuous : IfcLoopHeadToTail(SELF)

    // === DERIVE CLAUSES ===
    // Ne : IfcInteger := SIZEOF(EdgeList)
}
